tests/clpy_tests/core_tests/test_ndarray_scatter.py

```diff
@@ -112,7 +112,8 @@
 class TestScatterAddParametrized(unittest.TestCase):
 
     @testing.for_dtypes([numpy.float32, numpy.int32, numpy.uint32,
-                         numpy.uint64, numpy.ulonglong])
+                         # numpy.uint64, numpy.ulonglong])
```

Comment:
Please leave TODO on them. (and create issue to support)

Reply:
I improved the style of commenting out. They have been marked as TODO.

I left a note in Issue #30 and renamed it.